web/src/settings_sections.js

```diff
@@ -34,8 +34,7 @@ export function get_group(section) {
         case "bot-list-admin":
             return "org_bots";
 
-        case "user-list-admin":
-        case "deactivated-users-admin":
+        case "user-panel-admin":
```

Comment:
I think we should avoid renaming `user-list-admin` for now. Since we would include active and deactivated terms to denote different tab sections.